app/components/billing-status-badge.ts

```
@@ -22,10 +22,46 @@ export default class BillingStatusDisplayComponent extends Component<Signature>
+
+  get badgeType(): 'vip' | 'member' | 'discount-timer' | 'discount-timer-excluding-stage-2-completion' | 'free-weeks-left' | 'upgrade' {
+    if (this.shouldShowVipBadge) {
+      return 'vip';
+    }
+
+    if (this.shouldShowMemberBadge) {
+      return 'member';
+    }
+
+    if (this.activeDiscountForYearlyPlan && this.canSeeDiscountCountdown) {
+      // Stage 2 completion discounts are hidden behind a feature flag 
```

ESLint

[error] Delete `·`

(prettier/prettier)